src/main/java/com/commercetools/sync/inventory/utils/InventoryUpdateActionUtils.java

```diff
@@ -0,0 +1,106 @@
+package com.commercetools.sync.inventory.utils;
+
+import io.sphere.sdk.channels.Channel;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.InventoryEntryDraft;
+import io.sphere.sdk.inventory.commands.updateactions.ChangeQuantity;
+import io.sphere.sdk.inventory.commands.updateactions.SetExpectedDelivery;
+import io.sphere.sdk.inventory.commands.updateactions.SetRestockableInDays;
+import io.sphere.sdk.inventory.commands.updateactions.SetSupplyChannel;
+import io.sphere.sdk.models.Reference;
+
+import javax.annotation.Nonnull;
+import java.time.ZonedDateTime;
+import java.util.Optional;
+
+import static com.commercetools.sync.commons.utils.CommonTypeUpdateActionUtils.buildUpdateAction;
+
+/**
+ * This class provides static utility methods for building update actions related to inventories.
+ */
+public final class InventoryUpdateActionUtils {
+
+    private InventoryUpdateActionUtils() {
+        throw new AssertionError();
+    }
+
+    /**
+     * Compares the {@code quantityOnStock} values of a {@link InventoryEntry} and {@link InventoryEntryDraft}
+     * and returns a {@link Optional} of update action, which would contain the {@code "changeQuantity"}
+     * {@link UpdateAction}. If quantity on stock from {@code newEntry} is {@code null} or no update action is needed,
+     * for example in case where both the {@link InventoryEntry} and the {@link InventoryEntryDraft} have the same
+     * {@code quantityOnStock} values, the empty optional would be returned.
+     *
+     * @param oldEntry the inventory entry that should be updated
+     * @param newEntry the inventory entry draft which contains new quantity on stock.
+     * @return optional containing update action or empty optional if quantities on stock are indentical or quantity on stock
+     * from {@code newEntry} is null.
+     */
+    @Nonnull
+    public static Optional<UpdateAction<InventoryEntry>> buildChangeQuantityAction(@Nonnull final InventoryEntry oldEntry,
+                                                                                   @Nonnull final InventoryEntryDraft newEntry) {
+        final Long newQuantityOnStock = newEntry.getQuantityOnStock();
+        if (newQuantityOnStock == null) {
```

Comment:
@andrii-kovalenko-ct It is intended, but maybe can be improved. QuantityOnStock is required in CTP thus cannot be set to `null`,  `ChangeQuantity#of` update action takes primitive `long`. So `null` value has to be handled somehow. I'm not sure if should I treat it like 0L or like invalid data. I'll convert such `null` to 0L and then compare with oldValue (because quantityOnStock from `InventoryEntry` is not marked as Nullable, thus empty stock should result also in 0 value), what you think?